core-bundle/tests/EventListener/DataContainer/DisableBundleConfiguredSettingsListenerTest.php

```diff
@@ -17,16 +17,13 @@
 use Contao\CoreBundle\Tests\TestCase;
 use Contao\Image;
 use Doctrine\Common\Annotations\AnnotationReader;
-use PHPUnit\Framework\MockObject\MockObject;
 use Symfony\Contracts\Translation\TranslatorInterface;
-use Terminal42\ServiceAnnotationBundle\ServiceAnnotationInterface;
 
 class DisableBundleConfiguredSettingsListenerTest extends TestCase
 {
     public function testAnnotatedCallbacks(): void
     {
         $listener = $this->createListener();
-        $this->assertInstanceOf(ServiceAnnotationInterface::class, $listener);
```

Comment:
Why did you remove it? Shouldn't it be ensured that the interface is implemented otherwise the auto configuration won't work.

Reply:
PhpStorm told me that the assertion can be skipped:

<img width="744" alt="" src="https://user-images.githubusercontent.com/1192057/85846116-a4c3be80-b7a5-11ea-835b-d7caa3eec58f.png">

Reply:
Imho the annotation of EA is wrong as it does not know that it's a marker interface for the magic auto configuration.

Reply:
Agreed. I have re-added it in cde293e2137787a37497a3184edd32043b9213f2.